concordia/models.py

```diff
@@ -1,13 +1,14 @@
 from logging import getLogger
 from django.db import models
 from django.contrib.auth.models import User
+from django.conf import settings
 
 
 class UserProfile(models.Model):
     user = models.ForeignKey(User, on_delete=models.CASCADE)
     myfile = models.FileField(upload_to='profile_pics/')
 
-USE_POSTGRES = True
+USE_POSTGRES = settings.USE_POSTGRES
```

Comment:
Since this is only used in one place on the very next line, I wouldn't declare a separate variable for it.